src/index.js

```diff
@@ -26,7 +27,7 @@ class ExtractTextPlugin {
     if (isString(options)) {
       options = { filename: options };
     } else {
-      validateOptions(path.resolve(__dirname, '../schema/plugin.json'), options, 'Extract Text Plugin');
+      validateOptions(path.resolve(__dirname, './schema/plugin.json'), options, 'Extract Text Plugin');
```

Comment:
Why?

Reply:
./schema is not resolved correctly when I build the branch from source. Not sure how the build process is, but without changing the path it does not compile.

Reply:
This should be reverted and the schema folder moved back to where it was.